librz/core/cmd_descs/cmd_resize.yaml

```diff
@@ -32,6 +32,9 @@ commands:
     args:
       - name: oldbase
         type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: newbase
+        type: RZ_CMD_ARG_TYPE_RZNUM
+        optional: true
```

Comment:
Probably this command should be renamed and put somewhere else.. It is not related at all to "resize" functionality and I would expect it to be somewhere under `o` (for example `or`?)

Also, I believe the user should not have to pass the `oldbase` argument because Rizin should already be aware of that itself, without the user having to tell it.